dll/win32/ubtrfs/CMakeLists.txt

```diff
@@ -3,10 +3,9 @@ spec2def(ubtrfs.dll ubtrfs.spec)
 
 list(APPEND SOURCE
     ubtrfs.c
-    ubtrfs.rc
     ${CMAKE_CURRENT_BINARY_DIR}/ubtrfs.def)
 
-add_library(ubtrfs MODULE ${SOURCE})
```

Comment:
Is this change necessary?

Reply:
changed in e720186ed8beb765c75222249dfe7e6d78bb3ab8